{"name": "test_registration_001", "status": "failed", "statusDetails": {"message": "AssertionError: assert True == False", "trace": "self = <testCases.test_createAccount.Test_CreateAccount object at 0x0000015716CC1E90>\nsetup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"0c86b35b4965a9bb38dc107c76faf26d\")>\n\n    @pytest.mark.sanity\n    def test_registration_001(self, setup):\n        self.driver = setup\n        self.log.info(\"TestCase test_registration_001 is started\")\n        self.log.info(\"invoking the browser\")\n        self.driver.get(self.url)\n        self.log.info(\"opening URL\")\n        self.rp = UserRegistrationClass(self.driver)\n        self.rp.link_registration()\n        self.log.info(\"clicking on registration link\")\n        self.rp.firstname(\"Lukesh\")\n        self.log.info(\"entering firstname\")\n        self.rp.lastname(\"Ade\")\n        self.log.info(\"entering lastname\")\n        self.email = random_generator() + \"@gmail.com\"\n        self.rp.setEmail(self.email)\n        self.log.info(\"entering the email\")\n        self.rp.password(self.password)\n        self.log.info(\"entering the password\")\n        self.rp.confirm_password(self.password)\n        self.log.info(\"entering the confirm password\")\n        self.rp.submit_button()\n        self.log.info(\"clicking submit button\")\n        self.log.info(\"saving customer info\")\n        self.log.info(\"user_registration validation start\")\n        self.msg = self.driver.find_element(By.XPATH, \"/html/body/div[1]/main/div[1]/div[2]/div/div/div\").text\n        if 'Thank you for registering with Main Website Store.' in self.msg:\n            assert True == True\n            self.driver.save_screenshot(\".\\\\Screenshot\\\\test_registration_001_pass.png\")\n            self.log.info(\"test_registration_001 is pass\")\n        else:\n            self.driver.save_screenshot(\".\\\\Screenshot\\\\test_registration_001_fail.png\")\n            self.log.info(\"test_registration_001 is fail\")\n>           assert True == False\nE           assert True == False\n\ntestCases\\test_createAccount.py:51: AssertionError"}, "attachments": [{"name": "log", "source": "1237e01b-0f98-4feb-abb7-2b264ea0fd61-attachment.txt", "type": "text/plain"}], "start": 1691056400077, "stop": 1691056417353, "uuid": "606a2baa-5b3a-41b7-a140-9e582eb7c7c8", "historyId": "89373e9194b52c3e7a0aab752e1ca2cd", "testCaseId": "89373e9194b52c3e7a0aab752e1ca2cd", "fullName": "test_createAccount.Test_CreateAccount#test_registration_001", "labels": [{"name": "tag", "value": "sanity"}, {"name": "suite", "value": "test_createAccount"}, {"name": "subSuite", "value": "Test_CreateAccount"}, {"name": "host", "value": "DESKTOP-3ABMBD4"}, {"name": "thread", "value": "5780-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_createAccount"}]}